dmlproc/dmlprocessor.cpp

```diff
@@ -1112,6 +1110,9 @@ void PackageHandler::run()
         ml.logErrorMessage(message);
         result.result = DMLPackageProcessor::COMMAND_ERROR;
         result.message = message;
+
+        DistributedEngineComm* dec = DistributedEngineComm::instance(frm);
+        dec->Setup();
```

Comment:
The call to setup inside DEC works the majority of the time, but I hit a case where this point is reached without queries being runnable. Call setup again to avoid the edge case (only happens for dmlproc).